supertokens/humanise.go

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package supertokens
+
+import (
+	"fmt"
+	"math"
+)
+
+func formatOneDecimalFloat(n float64) (string, string) {
+	n = math.Floor(n*10) / 10
+	if float64(int(n)) == n {
+		if n == 1.0 {
+			return fmt.Sprintf("%d", int(n)), ""
+		} else {
+			return fmt.Sprintf("%d", int(n)), "s"
+		}
+	}
+	return fmt.Sprintf("%.1f", n), "s"
+}
+
+func HumaniseSeconds(t uint64) string {
```

Comment:
This doesn't seem to be used anywhere outside of this file. Maybe remove this?

Reply:
Removed